template/Cargo.toml

```diff
@@ -87,6 +87,7 @@ smoltcp = { version = "0.12.0", default-features = false, features = [
 # for more networking protocol support see https://crates.io/crates/edge-net
 #ENDIF embassy && wifi
 esp-wifi = { version = "0.12.0", default-features=false, features = [
```

Comment:
I think we can remove default-features = false here, we manually enable the two default features anyways :D